test/GoogleAnalyticsEventProcessor/buildRequestUrl_test.js

```diff
@@ -0,0 +1,87 @@
+goog.module('measurementLibrary.eventProcessor.testing.GoogleAnalyticsEventProcessor.buildRequestUrl');
+goog.setTestOnly();
+
+const GoogleAnalyticsEventProcessor = goog.require('measurementLibrary.eventProcessor.GoogleAnalyticsEventProcessor');
+
+describe('The `buildRequestUrl` method ' +
+    'of GoogleAnalyticsEventProcessor', () => {
+  /**
+   * Tests the `buildRequestUrl` function by passing in an API secret,
+   * measurement ID, and measurement URL to construct the event processor
+   * with and comparing the result of the function with the expected result.
+   * @param {string} apiSecret
+   * @param {string} measurementId
+   * @param {string} measurementUrl
+   * @param {string} expected
+   */
+  function checkRequestUrl(apiSecret, measurementId, measurementUrl, expected) {
+    const eventProcessor = new GoogleAnalyticsEventProcessor({
+      'api_secret': apiSecret,
+      'measurement_id': measurementId,
+      'measurement_url': measurementUrl,
+    });
+
+    expect(eventProcessor.buildRequestUrl())
+        .toBe(expected);
+  }
+
+  it('creates the correct api hit with default url', () => {
```

Comment:
Phrasing is a bit weird here, although tricky since the output is a URL but one of the inputs is too. 

Maybe "creates the correct query string when using the default measurement url" ?